metapool/tests/test_metapool.py

```diff
@@ -1259,6 +1374,20 @@ def test_add_syndna_pool1_noconcentration_exc(self):
             add_syndna(test_df, syndna_pool_number='pool1',
                        syndna_concentration=None)
 
+    def test_generate_override_cycles_value(self):
+        for fp, exp, adapter_length in self.runinfos:
+            obs = generate_override_cycles_value(fp, adapter_length)
+            # if an assertion fails, print the path of the most recently
+            # processed file for debugging purposes.
+            print(fp)
```

Comment:
This will be printed _all_ the time, not only when assertion fails, right? BTW I think that when a tests fails, it shows you why it failed and the message will have the values.

Reply:
@antgonza , this is @charles-cowart's new functionality that you merged into master on Wednesday: https://github.com/biocore/metagenomics_pooling_notebook/blob/95e78dcd16342967a6ef73666cee9b1a5e27969d/metapool/tests/test_metapool.py#L1289 .  All I did was back-merge it from master into the development branch.

Reply:
It will be printed all the time to stdout, but it will only be shown if the test fails, in which case the fp will be useful for debugging.

Reply:
I discussed it with Antonio and I mentioned I planned to revisit the logging library in the near future. I'll go ahead and issue a PR to remove the print statements in master immediately and you can merge the lastest master into this branch afterward or I can remove them after this branch has been merged. Whatever sounds easiest for everyone.

Reply:
@charles-cowart I say go ahead and change them :)

Reply:
I took the liberty of pushing directly to this branch, my apologies. The statements are removed! :)